### PR TITLE
Classification of Units added as a new view

### PR DESCRIPTION
**Pull request by Cheyans**
Same changes as #41 but rebased as requested. I have not tested my rebase as it was fairly trivial, but I trust you'll do so spooky.

### CODE REVIEW

**Comment by spooky** [comment]
No go - doesn't work. Seems that the changes pulled from a different pull request were not incorporated into the view. If I'll find some time, I'll try to merge them in, but if anyone else feels like doing it - that would be more than welcome.

**Comment by Cheyans** [comment]
It's because you just pushed a change 2 days ago, what did you expect to happen? It should be good now.

**Comment by spooky** [comment]
Actually, I'm referring to these changes:
https://github.com/spooky/unitdb/pull/37 - they were merged on Jan 20...
They're not incorporated in the gdi view.

File: package.json [hunk 1]
```diff
@@ -33,6 +33,8 @@
     "phantomjs-prebuilt": "2.1.4",
     "karma-phantomjs-launcher": "^1.0.0",
     "load-grunt-tasks": "^0.4.0",
+    "phantomjs": "^1.9.16",
+    "phantomjssmith": "^0.6.0",
```
Comment (Araamarmorth) [inline package.json]: Hi,

Just wondering why we need two versions of phantomjs here ?

Best,

Comment (spooky) [inline package.json]: most likely not :)

Comment (spooky) [inline package.json]: Well, just checked - phantomjssmith requires phandomjs - it uses it to create sprites. So the revised answer would be - yes, we do need it.

**Comment by Raistlfiren** [comment]
@spooky Any updates on getting this merged into master? I think this is a good addition to the project, if we can switch views. Thanks!

**Comment by spooky** [comment]
@Raistlfiren - see my [last comment](https://github.com/spooky/unitdb/pull/50#issuecomment-246998841) - this pull request needs more work as it breaks the current functionality. So, if anyone is up for that, it would be great.

File: README.md [hunk 2]
```diff
@@ -25,3 +25,19 @@ contributing
 All contributions are welcome, though I can't guarantee to pull all of them in. If you do want to contribute,
 please create a separate branch and a pull request for that. It'll be a bit easier for me to keep the repo tidy that way.  
 Thanks in advance.
+
+Running the Application
+-----------------------
+Necessary packages that need to be installed beforehand:
+
+> brew install pkgconfig
```
Comment (Cheyans) [inline README.md]: You may want to indicate that brew is OSX only

Comment (Raistlfiren) [inline README.md]: Good call, I will update that with my next commit. I am getting syntax errors when running the app right now that I need to troubleshoot.

**Comment by Raistlfiren** [comment]
@spooky I have updated the PR to include the changes in PR #37. If there is anything else that I might have missed, please ask. Thank you for all of your hard work spooky! :)

**Comment by spooky** [comment]
Awesome work, thanks, but..
this is still a no go. With the latest code updates, I'm not able to build a release version (just running the 'grunt' command needs to build a minified version of the site that's deployed to the gh-pages branch). Also, there's a bug - switching from the GDI view back to the default one does not work without a refresh.
I'll leave some comments in the pull request for the code itself.

File: Gruntfile.js [hunk 3]
```diff
@@ -176,14 +176,22 @@ module.exports = function (grunt) {
 
     // replace in files
     replace: {
-      stamp: {
+      homestamp: {
         src: ['<%= yeoman.app %>/views/home.html'],
```
Comment (spooky) [inline Gruntfile.js]: this is an array, instead of creating a separate task, just add the gdi.html to the list in the old task

File: .jshintrc [hunk 4]
```diff
@@ -8,7 +8,7 @@
   "eqeqeq": true,
   "immed": true,
   //"indent": 4,
-  "latedef": true,
```
Comment (spooky) [inline .jshintrc]: This setting was on for a reason, please fix the underlying issue that is reported by the linter instead of turning it off.

**Comment by Raistlfiren** [comment]
@spooky I have corrected the problems that you had with the fork yesterday, and I have went over the changes. Everything seems to be fine now. Switching between views works just fine, `grunt` is compiling correctly again(had to do with a package upgrade), and the warnings are fixed from jshinter.

One thing I did notice between the two controller methods are that the filters for factions, kinds, and tech get carried over from gdi to normal view but not vice versa. It is coming from this line - https://github.com/spooky/unitdb/pull/50/files#diff-40084eb42a8e52b0a5ff0264ef426c94R97 Do you want to keep that and change the normal view, or remove it so the user has a fresh slate when changing between views?

Thanks!

**Pull request merged**